<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=11"/>
<meta name="generator" content="Doxygen 1.9.4"/>
<meta name="viewport" content="width=device-width, initial-scale=1"/>
<title>IQ-TREE: IQ-TREE</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/searchdata.js"></script>
<script type="text/javascript" src="search/search.js"></script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr id="projectrow">
  <td id="projectalign">
   <div id="projectname">IQ-TREE<span id="projectnumber">&#160;2</span>
   </div>
   <div id="projectbrief">IQ-TREE phylogenetic software</div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.9.4 -->
<script type="text/javascript">
/* @license magnet:?xt=urn:btih:d3d9a9a6595521f9666a5e94cc830dab83b65699&amp;dn=expat.txt MIT */
var searchBox = new SearchBox("searchBox", "search",'Search','.html');
/* @license-end */
</script>
<script type="text/javascript" src="menudata.js"></script>
<script type="text/javascript" src="menu.js"></script>
<script type="text/javascript">
/* @license magnet:?xt=urn:btih:d3d9a9a6595521f9666a5e94cc830dab83b65699&amp;dn=expat.txt MIT */
$(function() {
  initMenu('',true,false,'search.php','Search');
  $(document).ready(function() { init_search(); });
});
/* @license-end */
</script>
<div id="main-nav"></div>
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
</div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

</div><!-- top -->
<div><div class="header">
  <div class="headertitle"><div class="title">IQ-TREE </div></div>
</div><!--header-->
<div class="contents">
<div class="textblock"><p ><a href="https://github.com/Cibiv/IQ-TREE/releases"><img src="https://img.shields.io/github/downloads/Cibiv/IQ-TREE/total.svg?style=social&amp;logo=github&amp;label=Download" alt="Github All Releases" style="pointer-events: none;" class="inline"/></a> <a href="https://anaconda.org/bioconda/iqtree"><img src="https://img.shields.io/conda/dn/bioconda/iqtree.svg?style=flag&amp;label=BioConda%20install" alt="BioConda downloads" style="pointer-events: none;" class="inline"/></a> <a href="https://travis-ci.org/bqminh/IQ-TREE"><img src="https://travis-ci.org/bqminh/IQ-TREE.svg?branch=master" alt="Build Status" style="pointer-events: none;" class="inline"/></a> <a href="https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html"><img src="https://img.shields.io/badge/License-GPL%20v2-blue.svg" alt="License: GPL v2" style="pointer-events: none;" class="inline"/></a></p>
<p >Efficient and versatile phylogenomic software by maximum likelihood <a href="http://www.iqtree.org">http://www.iqtree.org</a></p>
<h1><a class="anchor" id="autotoc_md0"></a>
Introduction</h1>
<p >The IQ-TREE software was created as the successor of IQPNNI and <a href="http://www.tree-puzzle.de">TREE-PUZZLE</a> (thus the name IQ-TREE). IQ-TREE was motivated by the rapid accumulation of phylogenomic data, leading to a need for efficient phylogenomic software that can handle a large amount of data and provide more complex models of sequence evolution. To this end, IQ-TREE can utilize multicore computers and distributed parallel computing to speed up the analysis. IQ-TREE automatically performs checkpointing to resume an interrupted analysis.</p>
<p >As input IQ-TREE accepts all common sequence alignment formats including PHYLIP, FASTA, Nexus, Clustal and MSF. As output IQ-TREE will write a self-readable report file (name suffix <code>.iqtree</code>), a NEWICK tree file (<code>.treefile</code>) which can be visualized by tree viewer programs such as <a href="http://tree.bio.ed.ac.uk/software/figtree/">FigTree</a>, <a href="http://dendroscope.org">Dendroscope</a> or <a href="http://itol.embl.de">iTOL</a>.</p>
<h1><a class="anchor" id="autotoc_md1"></a>
Key features of IQ-TREE</h1>
<ul>
<li><b>Efficient search algorithm</b>: Fast and effective stochastic algorithm to reconstruct phylogenetic trees by maximum likelihood. IQ-TREE compares favorably to RAxML and PhyML in terms of likelihood while requiring similar amount of computing time (<a href="http://dx.doi.org/10.1093/molbev/msu300">Nguyen et al., 2015</a>).</li>
<li><b>Ultrafast bootstrap</b>: An ultrafast bootstrap approximation (UFBoot) to assess branch supports. UFBoot is 10 to 40 times faster than RAxML rapid bootstrap and obtains less biased support values (<a href="http://dx.doi.org/10.1093/molbev/mst024">Minh et al., 2013</a>).</li>
<li><b>Ultrafast model selection</b>: An ultrafast and automatic model selection (ModelFinder) which is 10 to 100 times faster than jModelTest and ProtTest. ModelFinder also finds best-fit partitioning scheme like PartitionFinder (<a href="https://doi.org/10.1038/nmeth.4285">Kalyaanamoorthy et al., 2017</a>).</li>
<li><b>Phylogenetic testing</b>: Several fast branch tests like SH-aLRT and aBayes test (<a href="http://dx.doi.org/10.1093/sysbio/syr041">Anisimova et al., 2011</a>) and tree topology tests like the approximately unbiased (AU) test (<a href="http://dx.doi.org/10.1080/10635150290069913">Shimodaira, 2002</a>).</li>
</ul>
<p >The strength of IQ-TREE is the availability of a wide variety of phylogenetic models:</p>
<ul>
<li><b>Common models</b>: All <a href="http://www.iqtree.org/doc/Substitution-Models">common substitution models</a> for DNA, protein, codon, binary and morphological data with <a href="http://www.iqtree.org/doc/Substitution-Models#rate-heterogeneity-across-sites">rate heterogeneity among sites</a> and <a href="http://www.iqtree.org/doc/Substitution-Models#ascertainment-bias-correction">ascertainment bias correction</a> for e.g. SNP data.</li>
<li><b><a href="http://www.iqtree.org/doc/Complex-Models#partition-models">Partition models</a></b>: Allowing individual models for different genomic loci (e.g. genes or codon positions), mixed data types, mixed rate heterogeneity types, linked or unlinked branch lengths between partitions.</li>
<li><b>Mixture Models</b>: <a href="http://www.iqtree.org/doc/Complex-Models#mixture-models">fully customizable mixture models</a> and <a href="http://www.iqtree.org/doc/Substitution-Models#protein-models">empirical protein mixture models</a> and.</li>
<li><b>Polymorphism-aware models (PoMo)</b>: <a href="http://www.iqtree.org/doc/Polymorphism-Aware-Models">http://www.iqtree.org/doc/Polymorphism-Aware-Models</a></li>
</ul>
<h1><a class="anchor" id="autotoc_md2"></a>
IQ-TREE web service</h1>
<p >For a quick start you can also try the IQ-TREE web server, which performs online computation using a dedicated computing cluster. It is very easy to use with as few as just 3 clicks! Try it out at</p>
<p ><a href="http://iqtree.cibiv.univie.ac.at">http://iqtree.cibiv.univie.ac.at</a></p>
<h1><a class="anchor" id="autotoc_md3"></a>
User support</h1>
<p >Please refer to the <a href="http://www.iqtree.org/doc/">user documentation</a> and <a href="http://www.iqtree.org/doc/Frequently-Asked-Questions">frequently asked questions</a>. If you have further questions, feedback, feature requests, and bug reports, please sign up the following Google group (if not done yet) and post a topic to the</p>
<p ><a href="https://groups.google.com/d/forum/iqtree">https://groups.google.com/d/forum/iqtree</a></p>
<p ><em>The average response time is one working day.</em></p>
<h1><a class="anchor" id="autotoc_md4"></a>
Citations</h1>
<p >When using ModelFinder please cite:</p>
<ul>
<li>S. Kalyaanamoorthy, B.Q. Minh, T.K.F. Wong, A. von Haeseler, L.S. Jermiin (2017) ModelFinder: Fast model selection for accurate phylogenetic estimates. <em>Nat. Methods</em>, 14:587-589. <a href="https://doi.org/10.1038/nmeth.4285">https://doi.org/10.1038/nmeth.4285</a></li>
</ul>
<p >When performing tree reconstruction please cite:</p>
<ul>
<li>L.-T. Nguyen, H.A. Schmidt, A. von Haeseler, and B.Q. Minh (2015) IQ-TREE: A fast and effective stochastic algorithm for estimating maximum likelihood phylogenies. <em>Mol. Biol. Evol.</em>, 32, 268-274. <a href="https://doi.org/10.1093/molbev/msu300">https://doi.org/10.1093/molbev/msu300</a></li>
</ul>
<p >For the ultrafast bootstrap (UFBoot) please cite:</p>
<ul>
<li>D.T. Hoang, O. Chernomor, A. von Haeseler, B.Q. Minh, and L.S. Vinh (2017) UFBoot2: Improving the ultrafast bootstrap approximation. <em>Mol. Biol. Evol.</em>, in press. <a href="https://doi.org/10.1093/molbev/msx281">https://doi.org/10.1093/molbev/msx281</a></li>
</ul>
<p >When using posterior mean site frequency model (PMSF) please cite:</p>
<ul>
<li>H.C. Wang, B.Q. Minh, S. Susko, A.J. Roger (in press) Modeling site heterogeneity with posterior mean site frequency profiles accelerates accurate phylogenomic estimation. <em>Syst. Biol.</em> <a href="https://doi.org/10.1093/sysbio/syx068">https://doi.org/10.1093/sysbio/syx068</a></li>
</ul>
<p >When using partition models please cite:</p>
<ul>
<li>O. Chernomor, A. von Haeseler, B.Q. Minh (2016) Terrace aware data structure for phylogenomic inference from supermatrices. <em>Syst. Biol.</em>, 65:997-1008. <a href="https://doi.org/10.1093/sysbio/syw037">https://doi.org/10.1093/sysbio/syw037</a></li>
</ul>
<p >When using polymorphism-aware models please cite:</p>
<ul>
<li>D. Schrempf, B.Q. Minh, N. De Maio, A. von Haeseler, C. Kosiol (2016) Reversible polymorphism-aware phylogenetic models and their application to tree inference. <em>J. Theor. Biol.</em>, 407:362-370. <a href="https://doi.org/10.1016/j.jtbi.2016.07.042">https://doi.org/10.1016/j.jtbi.2016.07.042</a></li>
</ul>
<h3><a class="anchor" id="autotoc_md5"></a>
Credits and Acknowledgements</h3>
<p >Some parts of the code were taken from the following packages/libraries: <a href="http://www.libpll.org">Phylogenetic likelihood library</a>, <a href="http://www.tree-puzzle.de">TREE-PUZZLE</a>, <a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025808">BIONJ</a>, <a href="http://dx.doi.org/10.1093/bioinformatics/btg319">Nexus Class Libary</a>, <a href="http://eigen.tuxfamily.org/">Eigen library</a>, <a href="http://www.sprng.org">SPRNG library</a>, <a href="http://www.zlib.net">Zlib library</a>, <a href="http://www.cs.unc.edu/Research/compgeom/gzstream/">gzstream library</a>, <a href="http://www.agner.org/optimize/">vectorclass library</a>, <a href="https://www.gnu.org/software/gsl/">GNU scientific library</a>.</p>
<p >IQ-TREE was partially funded by the <a href="http://www.fwf.ac.at/">Austrian Science Fund - FWF</a> (grant no. I 760-B17 from 2012-2015 and and I 2508-B29 from 2016-2019) and the <a href="https://www.univie.ac.at/">University of Vienna</a> (Initiativkolleg I059-N). </p>
</div></div><!-- contents -->
</div><!-- PageDoc -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated by&#160;<a href="https://www.doxygen.org/index.html"><img class="footer" src="doxygen.svg" width="104" height="31" alt="doxygen"/></a> 1.9.4
</small></address>
</body>
</html>
